src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/MediationSuccessfulApplicantNotificationHandler.java

```diff
@@ -45,15 +52,59 @@ public String camundaActivityId(CallbackParams callbackParams) {
 
     private CallbackResponse notifyApplicant(CallbackParams callbackParams) {
         CaseData caseData = callbackParams.getCaseData();
-        notificationService.sendMail(
-            caseData.getApplicantSolicitor1UserDetails().getEmail(),
-            notificationsProperties.getNotifyApplicantLRMediationSuccessfulTemplate(),
-            addProperties(caseData),
-            String.format(REFERENCE_TEMPLATE, caseData.getLegacyCaseReference()));
+        Boolean isCarmEnabled = featureToggleService.isCarmEnabledForCase(caseData);
+        if (isCarmEnabled) {
+            String claimId = caseData.getCcdCaseReference().toString();
+            String referenceTemplate = String.format(REFERENCE_TEMPLATE, claimId);
+            MultiPartyScenario scenario = getMultiPartyScenario(caseData);
+            // LIP
+            if (caseData.isLipvLipOneVOne()) {
+                sendEmail(
+                    caseData.getApplicant1().getPartyEmail(),
+                    notificationsProperties.getNotifyLipClaimantSuccessfulMediation(),
```

Comment:
as commented in commons, lips have the same mail template, could we condense them to the same template rather than having one in claimant and one in respondent?

Reply:
fixed one template for both parties